modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -59,17 +57,14 @@ Type::
 Description::
 +
 --
-`FlowCollectorSpec` defines the desired state of the FlowCollector resource.  +
+Defines the desired state of the FlowCollector resource.  +
  +
- *: the mention of _"unsupported"_, or _"deprecated"_ for a feature throughout this document means that this feature is not officially supported by Red Hat. It might have been, for instance, contributed by the community and accepted without a formal agreement for maintenance. The product maintainers might provide some support for these features as a best effort only.
+ *: the mention of "unsupported", or "deprecated" for a feature throughout this document means that this feature is not officially supported by Red Hat. It might have been, for instance, contributed by the community and accepted without a formal agreement for maintenance. The product maintainers might provide some support for these features as a best effort only.
```

Comment:
any reason why we mention this multiple times?

Reply:
This is basically our boilerplate blurb whenever there are unsupported/deprecated items that are listed in the API documentation, so it appears at each one of those. Just because, if we put an asterisk and then refer to a single blurb on the page, the user has to go all the way to the bottom of the page to find the blurb then go back to where they were. So we just mention it at each point that its relevant.